src/std.rs

```diff
@@ -54,6 +54,14 @@ impl<T> List<T> {
             Err(_) => Err("<locked>"),
         }
     }
+
+    // Get the listener count by blocking
+    pub(crate) fn total_listeners_wait(&self) -> usize {
```

Comment:
Name this function `total_listeners` instead.

Reply:
total_listeners is already just above.